app/summary/directives/instructorSummary/scheduledCourses/scheduledCourses.js

```diff
@@ -40,21 +40,29 @@ let scheduledCourses = function ($rootScope) {
 			};
 
 			scope.mapDataToState = function(data) {
```

Comment:
Let's rename this function while we're here. Leave the campground cleaner than we found it. (The name is too generic.)